source/set-gitmodules.bat

```diff
@@ -6,6 +6,7 @@ if exist ".git" (
     git init
     git submodule add https://github.com/Tencent/rapidjson.git ./external/rapidjson
     git submodule add https://github.com/microsoft/libHttpClient.git ./external/libHttpClient
```

Comment:
you should change this to  [https://github.com/jasonsandlin/rapidjson.git](https://github.com/jasonsandlin/rapidjson.git) cause we need the patches i did for rapidjson for mem hooking which xsapi uses.

Reply:
The way the PlayFab SDK is using rapidjson shouldn't require those changes.  I have already reviewed the difference before and don't think we need them.  Either way, there is a seperate task to validate that we aren't allocating anywhere and we can always switch at that point if we find out it is needed.